resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnAllocatorSuite.scala

```diff
@@ -592,11 +601,13 @@ class YarnAllocatorSuite extends SparkFunSuite with Matchers with BeforeAndAfter
 
   test("SPARK-26269: YarnAllocator should have same blacklist behaviour with YARN") {
     val rmClientSpy = spy(rmClient)
+    val yarnClientSpy = spy(yarnClient)
```

Comment:
As I see this spy is not needed it is enough to pass `yarnClient ` when `createAllocator ` is called.